backend/server.ts

```diff
@@ -18,9 +19,47 @@ app.get('/', (req, res) => {
 })
 
 //get all users
-app.get("/api/users", async (req: Request, res: Response) => {
+app.get("/users", async (req: Request, res: Response) => 
+{
   const users = await User.find({});
   res.send(users)
 })
 
+//gets user by id
+app.get("/user/:id", async (req: Request, res: Response) => 
+{
+  const user = await User.findOne({recipeName: req.params.id})
```

Comment:
We want to find the User based on "_id", not "recipeName". You can either use the function findById or replace the "recipeName" with "_id"